contingency/contingency-api/src/main/java/com/powsybl/contingency/json/IdentifierDeserializer.java

```diff
@@ -44,6 +46,11 @@ public NetworkElementIdentifier deserialize(JsonParser parser, DeserializationCo
                 case "identifier":
                     identifier = parser.nextTextValue();
                     break;
+                case "identifiableType":
+                    JsonUtil.assertLessThanOrEqualToReferenceVersion(CONTEXT_NAME, "identifiableType",
+                            (String) deserializationContext.getAttribute("version"), "1.0");
+                    parser.nextToken();
+                    break;
```

Comment:
Was it a mistake to put it here instead of in `IdentifierContingencyListDeserializer`?

Reply:
ok I mistook different type